Source/WebKit/UIProcess/API/APINavigationClient.h

```diff
@@ -116,7 +118,7 @@ class NavigationClient {
     virtual void processDidBecomeResponsive(WebKit::WebPageProxy&) { }
     virtual void processDidBecomeUnresponsive(WebKit::WebPageProxy&) { }
 
-    virtual RefPtr<Data> webCryptoMasterKey(WebKit::WebPageProxy&) { return nullptr; }
+    virtual RefPtr<Data> webCryptoMasterKey(WebKit::WebPageProxy&, DefaultMasterKeyAsFallback = DefaultMasterKeyAsFallback::No) { return nullptr; }
```

Comment:
The parameter DefaultMasterKeyAsFallback makes the flow hard to follow.  I think this entire function should be removed in favor of only asking the WebsiteDataStoreClient.